docs/rules/alphabetical-paths.md

```diff
@@ -0,0 +1,102 @@
+#  Ensure that required paths are in alphabetical order (alphabetical-paths)
+
+## Rule Details
+
+This rule checks whether the required paths in `define` and `require` statements are order alphabetically or not.
+
+### Options
+
+This rule takes two arguments. First argument is optionoal and can be "fullpath", or "basename". If it is "fullpath" then whole path strings are checked for right order. If "basename", then just basename parts of required paths are checked. The default is "fullpath".
+
+The second argument is also optional and will be used only when the first argument is manually set to "fullpath". It can be "always", or "never". If it is "always" then it checks at first just dirnames of paths and then, if they are equal, it checks basenames. If it is "never" then it just checks entire paths. The default is "always".
```

Comment:
Thanks for the comments! Yep it is confusing...after few hours I'm also a bit confused ;) And I absolutely agree, that options must be absolutely clear. I'll rewrite it a bit in next few days, can't think about that right now. I'll check all use cases again and will try to find the simplest solution with the clearest api.

It's really cool to have such fast response :+1: